tests/integration_tests/test_qwen_embedding_function.py

```
@@ -0,0 +1,215 @@
+        # Test with timeout and max_retries (api_base uses default)
+        ef = QwenEmbeddingFunction(
+            model_name="text-embedding-v1",
+            timeout=30,
+            max_retries=3
+        )
+
+        test_doc = "测试文档用于嵌入"
+        embeddings = ef(test_doc)
+        assert len(embeddings) == 1
+        assert len(embeddings[0]) > 0
+        print(f"   Embedding generated successfully with custom parameters")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)